lib/api/client-commands/axeRun.js

```diff
@@ -20,6 +20,7 @@
  * @syntax browser.axeRun('body')
  * @param {string} selector The CSS selector used to locate the element.
  * @param {object} options Object containing rules configuration to use when performing the analysis
+ * @see {@link https://www.deque.com/axe/core-documentation/api-documentation/#options-parameter}
```

Comment:
```suggestion
 * @see https://www.deque.com/axe/core-documentation/api-documentation/#options-parameter
```

We don't need the `@link` tag